x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/alerts/basic_license_essentials_tier/alert_status/alert_status.ts

```diff
@@ -262,75 +214,6 @@ export default ({ getService }: FtrProviderContext) => {
           );
           expect(everyAlertWorkflowStatusUpdatedAtExists).to.eql(true);
         });
-        // This fails and should be investigated or removed if it no longer applies
-        it.skip('should be able to close alerts with t1 analyst user', async () => {
```

Comment:
Was skipped because it was failing. Moved to ESS specific file and updated roles used for testing. Now unskipped. Need matching tests for prebuilt serverless roles (see issues I created to track in description).